assets/src/application/ui/layout/entityList/filterBar/filterState/useEntityListFilterStateManager.ts

```diff
@@ -20,66 +20,81 @@ const useEntityListFilterStateManager = <SortBy = BasicSortBy>(defaultSortBy: So
 	};
 	const [state, dispatch] = useReducer(getReducer<SortBy>(), initialState);
 
-	const setSortBy: FSM['setSortBy'] = (sortBy) => {
-		dispatch({
-			type: 'SET_SORT_BY',
-			sortBy,
-		});
-	};
+	const setSortBy: FSM['setSortBy'] = useCallback(
+		(sortBy) => {
+			dispatch({
+				type: 'SET_SORT_BY',
+				sortBy,
+			});
+		},
+		[dispatch]
```

Comment:
why `dispatch` is set as dep, but not `sortBy`?

Reply:
`sortBy` is not in outer scope 😄